diff --git a/dist/evaluateFromFileSystem.js b/dist/evaluateFromFileSystem.js
index 8adee8897407a9d496a31f315bd1341e28e0ab8a..6de8dd8a5b72700f55004ba610f36e9a0d7dc638 100644
--- a/dist/evaluateFromFileSystem.js
+++ b/dist/evaluateFromFileSystem.js
@@ -33,7 +33,10 @@ function createLinker(parentModulePath, fsModule, rootDir, extraGlobals) {
         const absPath = path_1.default.join(path_1.default.dirname(parentModulePath), specifier);
         if (!getFromSourceModules(specifier, fsModule, rootDir)) {
             logging_1.log(`Using external require for ${specifier} from ${parentModulePath}`);
-            return require(specifier);
+
+            const absPath = require.resolve(specifier, {paths: [rootDir]})
+
+            return require(absPath);
         }
         logging_1.log(`Linking ${parentModulePath} to asset ${specifier}`);
         return evaluateFromFileSystem(absPath, fsModule, rootDir, extraGlobals);