diff --git a/index.js b/index.js
index 4292de776394e99437b874945e8c07b8b67f6c11..70c59341b8ad387943d5e9d54f0a36a3e62ea78e 100644
--- a/index.js
+++ b/index.js
@@ -74,11 +74,15 @@ Script.prototype.runInContext = function (context) {
     }
     
     forEach(Object_keys(context), function (key) {
-        win[key] = context[key];
+        try {
+            win[key] = context[key];
+        } catch (e) {}
     });
     forEach(globals, function (key) {
         if (context[key]) {
-            win[key] = context[key];
+            try {
+                win[key] = context[key];
+            } catch (e) {}
         }
     });
     